### PR TITLE
[libpqxx] Upgrade up to 7.6.0

### PR DESCRIPTION
**Pull request by alexander-smyslov**
**Describe the pull request**

- #### What does your PR fix?  
  Upgrade 7.6.0.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
 all Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes 

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/libpqxx/fix-build-with-vs2017.patch [hunk 1]
```diff
@@ -1,275 +0,0 @@
-diff --git a/include/pqxx/binarystring.hxx b/include/pqxx/binarystring.hxx
```
Comment (JonLiu1993) [inline ports/libpqxx/fix-build-with-vs2017.patch:1]: @alexander-smyslov ,Why delete this patch? I checked the source and found that it was not applied to the latest version

File: ports/libpqxx/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "libpqxx",
+  "version": "7.6.0",
```
Comment (JonLiu1993) [inline ports/libpqxx/vcpkg.json:3]:
```suggestion
  "version-semver": "7.6.0",
```

File: ports/libpqxx/vcpkg.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  "port-version": 2,
```
Comment (JonLiu1993) [inline ports/libpqxx/vcpkg.json:4]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/libpqxx/fix-build-with-vs2017.patch (hunk 1 above)
Comment (alexander-smyslov) [reply ports/libpqxx/fix-build-with-vs2017.patch:1]: This change was merged https://github.com/jtv/libpqxx/pull/406

Comment (JonLiu1993) [reply ports/libpqxx/fix-build-with-vs2017.patch:1]: Thanks for your clarification

File: versions/l-/libpqxx.json [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "bbbc587d55d5d1ee059e417b045dfc82ca4897c2",
+      "version-semver": "7.6.0",
+      "port-version": 0
+    },
+    {
+      "git-tree": "49a01ea25c7e6348079ca38cfbd46e2c4ba26eb2",
+      "version": "7.6.0",
+      "port-version": 2
+    },
```
Comment (JonLiu1993) [inline versions/l-/libpqxx.json:12]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**